node/src/restarter.rs

```diff
@@ -48,7 +48,16 @@ impl NodeRestarter {
         let mut committee = committee.clone();
 
         let mut handles = Vec::new();
-        let mut worker_network = PrimaryToWorkerNetwork::default();
+        let network = anemo::Network::bind("127.0.0.1:0")
+            .server_name("narwhal")
+            .private_key(
+                NetworkKeyPair::generate(&mut rand::rngs::OsRng)
```

Comment:
Maybe add a short explanation for why we generate a random private key here and why it's ok?

Reply:
yeah i'll add a comment